app/controllers/auth/signUp.js

```diff
@@ -0,0 +1,38 @@
+const { User } = require('../../models');
```

Comment:
Lest have all file names on `spine-case`, so... `sign-up.js`

Reply:
Fixed it